lib/tyrant/railtie.rb

```diff
@@ -1,17 +1,18 @@
 module Tyrant
   class Railtie < Rails::Railtie
-      require "warden"
+    require "warden"
 
-      # DISCUSS: it will be configurable what user class etc. and might be moved to Ops.
-      # DISCUSS: this should actually use Tyrant::Session to log in and out user?
-      config.app_middleware.use Warden::Manager do |config|
-        Warden::Manager.serialize_into_session do |user|
-          user.id
-        end
+    # DISCUSS: this should actually use Tyrant::Session to log in and out user?
+    config.app_middleware.use Warden::Manager do |config|
```

Comment:
Absolutely valid point, but I'd rather let users tell us the user class somehow, and also let them override the finding, e.g.

```ruby
config.tyrant.user = {class: User, find: ->(record) { User.find_by(id: record[:id]) } 
```

This way, we don't need all that Rails magic and could theoretically reuse that for other frameworks, e.g. Lotus. Thoughts?